src/components/Composer.vue

```diff
@@ -26,17 +29,19 @@
 				label="label"
 				track-by="email"
 				:multiple="true"
+				:placeholder="t('mail', 'Contact or email address …')"
+				:show-no-options="false"
 				@keyup="onInputChanged"
 				@tag="onNewToAddr"
 				@search-change="onAutocomplete"
 			/>
-			<a v-if="!showCC" href="#" @click.prevent="showCC = true">
-				{{ t('mail', '+ CC/BCC') }}
+			<a v-if="!showCC" class="transparency" href="#" @click.prevent="showCC = true">
+				{{ t('mail', '+ Cc/Bcc') }}
```

Comment:
It was uppercase intentionally https://github.com/nextcloud/mail/pull/2071

Reply:
So I changed it because both Gmail and Microsoft Outlook use "Cc" and "Bcc", and full-on uppercase just looks shouty.

And actually Apple Mail uses "Copy" and "Blind copy", which I would change it to instead as it is a lot less cryptic. :)

Reply:
cc @rakekniven

Reply:
@jancborchardt I checked older KMail and Thunderbird and Wikipedia.
Please change to other resources as well.

Reply:
@rakekniven Wikipedia also uses "Cc" and "Bcc" (as well as of course explaining it stands for "Carbon copy" and "Blind carbon copy") https://en.wikipedia.org/wiki/Blind_carbon_copy

By other occurences do you mean in the Mail app, or is this elsewhere? Thanks for your help! :)